redux/auth/saga.test.ts

```diff
@@ -1,40 +1,126 @@
-import { delay, put } from 'redux-saga/effects';
+import { put } from 'redux-saga/effects';
 
-import { loginSaga } from './saga';
-import { DOMAIN_NAME, LOGIN, LOGIN_FAILURE, LOGIN_SUCCESS } from './types';
+import { loginSaga, signupSaga } from './saga';
```

Comment:
Can you add a test for `loginWithFacebookSaga`?

Reply:
I was afraid whether you will mention it :laughing: . Yea I will do it